src/main/java/com/zendesk/maxwell/MaxwellConfig.java

```diff
@@ -102,8 +102,8 @@
 
 	public String rabbitmqUser;
 	public String rabbitmqPass;
-	public int rabbitmqHost;
-	public String rabbitmqPort;
+	public String rabbitmqHost;
+	public int rabbitmqPort;
```

Comment:
This commit should have been squashed with the previous commit [8d1a31](https://github.com/zendesk/maxwell/pull/921/commits/8d1a31b20c5f26ef1f5fdd78cf662945f68e7403).